OJP/OJP_Locations.xsd

```diff
@@ -51,26 +51,26 @@
 	<xs:annotation>
 		<xs:documentation>========================================== Additional declarations ==========================================</xs:documentation>
 	</xs:annotation>
-	<xs:complexType name="InitialPlaceInputStructure">
+	<xs:complexType name="InitialLocationInputStructure">
 		<xs:sequence>
 			<xs:element name="Name" type="xs:string" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Name of the place object which is looked after. This is usually the user's input. If not given, the name of the resulting place objects is not relevant.</xs:documentation>
+					<xs:documentation>Name of the LOCATION / PLACE object which is looked after. This is usually the user's input. If not given, the name of the resulting location / place objects is not relevant.</xs:documentation>
```

Comment:
"looked *for*" since we're searching for an object - "looked *after*" would mean that we want to take care of it.

Reply:
done